tools/config.json

```diff
@@ -4,16 +4,15 @@
   "stage_url": "https://dist.apache.org/repos/dist/dev/incubator/openwhisk",
   "release_url": "https://dist.apache.org/repos/dist/release/incubator/openwhisk",
   "versioning": {
-    "version": "0.11.0-incubating",
-    "pre_release_version": "rc2"
+    "version": "3.19.0-incubating",
+    "pre_release_version": "rc1"
   },
   "RepoList": [
-      "openwhisk-composer"
+    "openwhisk-client-js"
   ],
-  "openwhisk_composer": {
-    "name": "OpenWhisk Composer",
```

Comment:
the config file should have the `name` property as on line 14.

Reply:
It is good to have, but I do think it is mandatory for the release build and push in Travis.